Formula/s/sile.rb

```diff
@@ -193,27 +193,12 @@ def install
       --disable-silent-rules
       --with-system-luarocks
       --with-system-lua-sources
-      --disable-embeded-resources
```

Comment:
Doesn't this mean we should get rid of all of the `resource` blocks above? Are these included in the tarball?

Reply:
This option in dropped here because it is the default value anyway and so there is no point in setting it. It just clutters the build and makes it less clear what is going on.

The embedded resources system is completely orthogonal to Homebrew's "resources" (actually dependencies). This is for bundling all of the SILE core files into the final binary.